core/src/main/java/com/amplifyframework/storage/operation/StorageDownloadFileOperation.java

```diff
@@ -67,5 +67,17 @@ protected StorageDownloadFileOperation(
     public void setOnSuccess(@Nullable Consumer<StorageDownloadFileResult> onSuccess) {
         super.setOnSuccess(onSuccess);
     }
+
+    /**
+     * Request will be null if the operation is returned by
+     * {@link com.amplifyframework.storage.s3.AWSS3StoragePlugin} getTransfer api.
+     *
+     * @return the request object
+     */
+    @Override
```

Comment:
@palpatim Please review for public api changes.

Reply:
Public API changes approved.